main.go

```diff
@@ -204,7 +204,7 @@ func main() {
 	}
 
 	// Setup all Controllers
-	if err = toolchaincluster.NewReconciler(
+	if err = toolchainclustercache.NewReconciler(
 		mgr,
 		namespace,
 		crtConfig.ToolchainCluster().HealthCheckTimeout(),
```

Comment:
The same - update also the error message: https://github.com/codeready-toolchain/member-operator/blob/b6f420e51764c27dd044563d0497a8e2af27850e/main.go#L212

Reply:
Updated it